src/pages/Schedule/SchedulePage.tsx

```diff
@@ -1,136 +0,0 @@
-import Bugsnag from "@bugsnag/js";
```

Comment:
removed `SchedulePage` as it no longer used. 
https://github.com/sparkletown/sparkle/pull/1302/commits/a6cd04b7351b3b004d58205991ae7fd7e65d4ead